### ISSUE TITLE
Asistencia de Alumnos

### ISSUE THREAD

**Issue by Nikelondeon** [proyectoalba/alba2#12]
De carga semanal. Volcando en observaciones datos (*)informales.
Pensar la posibilidad de utilizar las notebooks o tablets (alguna app o extensión para Chrome por ejemplo) para actualización inmediata de la asistencia en el recorrido por las aulas (se hace todos los días, al iniciar el turno), 

(*) Elección multiple, para registrar si falta de manera consecutiva, una enfermedad de reposo temporario / vacaciones de la familia/ accidente en ed, fisica. etc.

**Comment by lucianobaraglia** [comment]
Me gusta la idea de las observaciones.
¿Te parece que a cada inasistencia se le pueda poner una observación? 
Esto al margen de que las inasistencias se definen como justificadas/injustificadas.

Lo de la aplicación para celulares está bueno, es más, lo pensábamos por el lado de los padres/tutores, para que les lleguen notificaciones/avisos, vean cómo van las notas de sus hijos, si faltaron, etc.

Igualmente es para analizar, porque tenemos que pensar si realmente va a ser utilizado.
¿Vos creés que en la mayoría de los institutos está la posibilidad de que tengan tablets y que dispongan de personal para que vayan aula por aula tomando notas?

La idea me encanta.

**Comment by Nikelondeon** [comment]
Sí, poder hacer observaciones de cada inasistencia es una buena solución.

Yo pienso más en las Apps para la Escuela que para la Familia. Sí le facilitás al preceptor la tarea de la carga de datos, es un paso más para el éxito de la aplicación.
Acordate que las Escuelas Secundarias tienen las Netbooks, además el Ministerio de Educación de la Nación provee del "Plan Mejoras" a las Escuelas donde pueden solicitar equipamiento específico.

**Comment by lucianobaraglia** [comment]
Excelente! :smiley:

**Comment by chichino2014** [comment]
Considero que los preceptores debes efectuar la carga diariamente, pues si el alumno falta en forma injustificada, ese mismo dia debe llamar al tutor para reportarle la situacion sobre todo por el tema de la seguridad y tranquilidad, tanto para el tutor como para el colegio
No importa tanto que la info le llegue al tutor via web, en el sentido del tema que nos ocupa, la falta, con solo enviarle un mensaje de texto al tutor en caso de no poder ubicarlo por voz ya se desencadena el efecto buscado

**Comment by hectorsanchez** [comment]
Para facilitar la carga la preceptor, serviría tener algunos textos pre cargados?
